geom/type_multi_point.go

```diff
@@ -9,14 +9,13 @@ import (
 // zero value is the empty MultiPoint (i.e. a collection of zero points) with
 // 2D coordinates type. It is immutable after creation.
 type MultiPoint struct {
-	seq   Sequence
-	empty BitSet
```

Comment:
`BitSet` is no longer used by anything, so I'll remove it from the codebase. Even though that increase the surface area of the API that is involved in the breaking change, I think that's better than leaving unused code around.

Reply:
Removed in a22a463d176504a3ed604b912d7453331fa6be4d